README.md

```diff
@@ -7,3 +7,6 @@ Changes made in: 2022.02.08	(20220208.1612)
 
 
 @20220208.2022 --> Studying the implications and interactions between different branches and commits in GIT (Figuring GIT data structures :-) )
+
+
+@20220208.2131 --> Branch created for pull request ( branch2pr )
```

Comment:
Descritive README file created, and correctly edited with the relevant information.
Licence (MIT License) file added and correctly edited to be right for this project.

Static html static files for README.md and MIT.md added so that the content can be properly viewed with this "application" hosted on a webserver (they are manualy generated, right now, using:  markdown <MD-file>  >  <HTML-file>.

Project deployed through:  **_git clone https://github.com/Microverse-ajgomes/Hello-Microverse.git_**  on the target webserver  (https://projects.microverse.datanet-pt.net/Hello-Microverse/), and thoroughly tested.